azurelinuxagent/common/protocol/wire.py

```diff
@@ -762,10 +762,7 @@ def _fetch_response(self, uri, headers=None, use_proxy=None, max_retry=None, ok_
         except (HttpError, ProtocolError, IOError) as error:
             msg = "Fetch failed: {0}".format(error)
             logger.warn(msg)
-            report_event(op=WALAEventOperation.Download,
```

Comment:
This should not be reported as a download error. Added event HttpGet for this kind of errors instead.